### PR TITLE
Apply patches to K8s packages

### PR DESCRIPTION
**Pull request by pburkholder**
## Changes proposed in this pull request:

- Use YAML anchor/alias to DRY (if that's a thing we can do)
- Apply proper fix for https://usn.ubuntu.com/4309-1/ -- vim
- Apply fix for https://usn.ubuntu.com/4293-1/ -- libarchive
- Apply fix for https://usn.ubuntu.com/4298-1/ -- sqlite
- Apply fix for https://usn.ubuntu.com/4305-1/ -- ICU (international components for unicode)
- Apply fix for https://usn.ubuntu.com/4333-1/ -- python

## security considerations

Enhances security by applying patches

### CODE REVIEW

File: templates/k8s-jobs.yml [hunk 1]
```diff
@@ -6,9 +6,11 @@ instance_groups:
   - name: pre-start-script
     release: os-conf
     properties:
-      script: |
+      script: &apt_patch |
         #!/bin/bash
-        apt-get -y remove rsync vim-tiny
+        apt-get update
+        apt-get -y remove rsync vim-tiny vim-gui-common python2.7-minimal python3.5-minimal
+        apt-get -y upgrade vim vim-common vim-runtime libarchive13 libsqlite3-0 sqlite3 libicu55 python2.7 python3.5
```
Comment (ccostino) [inline templates/k8s-jobs.yml:13]: We still need `python2.7` even though it's at end of life?  I'm guessing it's for one of the dependencies in the environment?

File: templates/k8s-jobs.yml [hunk 2]
```diff
@@ -6,9 +6,11 @@ instance_groups:
   - name: pre-start-script
     release: os-conf
     properties:
-      script: |
+      script: &apt_patch |
```
Comment (ccostino) [inline templates/k8s-jobs.yml:9]: As I'm still unfamiliar with the setup of a lot of these things, this caught my eye: what is the difference between this `&apt_patch` value and the `*apt_patch` values below?

Comment (apburnes) [reply templates/k8s-jobs.yml:9]: It's a yaml "anchor" to assign and reuse "variables". https://blog.daemonl.com/2016/02/yaml.html

Comment (pburkholder) [reply templates/k8s-jobs.yml:9]: @ccostino : It was helpful to me to use `yaml2json` on the file and ensure I got the expected output.

File: templates/k8s-jobs.yml (hunk 1 above)
Comment (pburkholder) [reply templates/k8s-jobs.yml:13]: It seemed safer to patch python2 on the system than to yank it and see stuff go sideways.

File: templates/k8s-jobs.yml (hunk 2 above)
Comment (ccostino) [reply templates/k8s-jobs.yml:9]: Thank you both, I appreciate it!

**Comment by pburkholder** [comment]
@cloud-gov/platform-ops :wave: Still need a review approval here to merge this. Thanks. P

**Comment by pburkholder** [comment]
@cloud-gov/platform-ops  -- I'm not authorized to merge this, so could one of y'alls merge it in? -- Thanks!

File: templates/k8s-jobs.yml (hunk 1 above)
Comment (mxplusb) [reply templates/k8s-jobs.yml:13]: As a sidebar, `vim` installs `vim-common` and `vim-runtime` as it's a metapackage.

**Comment by ChrisMcGowan** [comment]
So 2nd approval, adding a label, nor assignment allows me to merge this.

@mxplusb  - Can you merge this?

I wonder if it has something to do with the new security groups after we moved this out of 18F org

**Pull request merged**